docs/config/index.md

```diff
@@ -153,13 +153,14 @@ export default defineConfig({
 
 ### environment
 
-- **Type:** `'node' | 'jsdom' | 'happy-dom'`
+- **Type:** `'node' | 'jsdom' | 'happy-dom' | 'edge'`
```

Comment:
```suggestion
- **Type:** `'node' | 'jsdom' | 'happy-dom' | 'edge-runtime'`
```

`edge` feels too general without knowing the context, maybe better to be `edge-runtime`?